pkgs/tools/audio/alsaequal/default.nix

```diff
@@ -0,0 +1,88 @@
+{ stdenv, fetchurl
+, alsaLib, caps
+, alsaUtils
+}:
+
+stdenv.mkDerivation rec {
+  name = "alsaequal";
+  version = "0.6";
+
+  src = fetchurl {
+    url = "https://thedigitalmachine.net/tools/alsaequal-${version}.tar.bz2";
+    sha256 = "1w3g9q5z3nrn3mwdhaq6zsg0jila8d102dgwgrhj9vfx58apsvli";
+  };
+
+  buildInputs = [ alsaLib ];
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+
+  patches = [ ./makefile.patch ./false_error.patch ./caps_9.x.patch ];
```

Comment:
What do these patches fix? Are there any upstream references?

Reply:
These patches are taken from arch linux AUR package "alsaequal".

- "makefile.patch" adds executable permissions to resulting libraries and changes their destination directory from "usr/lib/alsa-lib" to "lib/alsa-lib" to better align with nixpkgs filesystem hierarchy.

- "false_error.patch" fixes control port check, which resulted in false error.

- "caps_9.x.patch" is due to name change of an "Eq" to "Eq10" method in version 9 of caps library.

Reply:
```suggestion
  # Borrowed from Arch Linux's AUR
  patches = [
    # Adds executable permissions to resulting libraries and changes their destination directory from "usr/lib/alsa-lib" to "lib/alsa-lib" to better align with nixpkgs filesystem hierarchy.
    ./makefile.patch
    # Fixes control port check, which resulted in false error.
    ./false_error.patch
    # Fixes name change of an "Eq" to "Eq10" method in version 9 of caps library.
    ./caps_9.x.patch
  ];
```